src/event/event_config.h

```diff
@@ -133,7 +133,7 @@
 #	define EV_ONESHOT				0x0010
 #	define EV_CLEAR					0x0020
 #	define EV_DISPATCH				0x0080
-#	define EV_UDATA_SPECIFIC		0x0100
+#	define EV_UDATA_SPECIFIC			0x0000
```

Comment:
should this really be defined at all ? I would have thought the `#ifdef EV_UDATA_SPECIFIC` below would have done the right thing if it is not